Library/Contributions/manpages/brew.1.md

```diff
@@ -305,10 +305,15 @@ Note that these flags should only appear after a command.
     Display all locally available formulae for brewing (including tapped ones).
     No online search is performed if called without arguments.
 
-  * `search`, `-S` <text>|/<text>/:
+  * `search`, `-S` <tap>:
+    Display all formulae in a <tap>, even if not yet tapped.
+    <tap> is of the form <user>/<repo>, e.g. `brew search homebrew/dupes`.
```

Comment:
This works for both tapped and untapped taps. Also, the substring search below works for untapped taps.